core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -119,39 +117,36 @@ protected override void OnResume()
 
             _restarted = true;
 
-            /*_customTabsActivityManager = new CustomTabsActivityManager(this);
-            _customTabsActivityManager.CustomTabsServiceConnected += delegate { _customTabsActivityManager.LaunchUrl(_requestUrl); };*/
             string chromePackageWithCustomTabSupport = GetChromePackageWithCustomTabSupport(ApplicationContext);
 
             if (string.IsNullOrEmpty(chromePackageWithCustomTabSupport))
             {
-                string chromePackage = GetChromePackage();
-                if (string.IsNullOrEmpty(chromePackage))
-                {
-                    const string chromeNotInstalledMessage = " Chrome is not installed on the device, cannot proceed with authentication";
-                    throw CoreExceptionFactory.Instance.GetClientException(
-                        CoreErrorCodes.ChromeNotInstalledError, 
-                        chromeNotInstalledMessage);
-                }
-
-                Intent browserIntent = new Intent(Intent.ActionView);
-                browserIntent.SetData(Uri.Parse(_requestUrl));
-                browserIntent.SetPackage(chromePackage);
+                Intent browserIntent = new Intent(Intent.ActionView, Uri.Parse(_requestUrl));
                 browserIntent.AddCategory(Intent.CategoryBrowsable);
+
+                CoreLoggerBase.Default.Info(
+                    "Browser with custom tabs package not available. " +
+                    "Launching with alternate browser. See https://aka.ms/msal-net-system-browsers for details.");
+
                 try
                 {
                     StartActivity(browserIntent);
                 }
                 catch (ActivityNotFoundException ex)
                 {
-                    const string chromeDisabledMessage = "Chrome is disabled on the device, cannot proceed with authentication ";
                     throw CoreExceptionFactory.Instance.GetClientException(
-                        CoreErrorCodes.ChromeDisabledError, 
-                        chromeDisabledMessage + ex.InnerException);
+                           CoreErrorCodes.AndroidActivityNotFound,
+                           CoreErrorMessages.AndroidActivityNotFound, ex);
```

Comment:
If the user does not have any browsers installed OR chrome is disabled on the device, we will throw here.